schemas/rum/resource-schema.json

```diff
@@ -223,6 +223,30 @@
                 }
               },
               "readOnly": true
+            },
+            "graphql":{
+              "type": "object",
+              "description": "GraphQL requests parameters",
+              "required": ["operationType", "query"],
+              "properties": {
+                "operationType": {
+                  "type": "string",
+                        "description": "Type of the GraphQL operation",
+                  "enum": ["query", "mutation", "subscription"],
```

Comment:
weird formatting 🤔
It seems that the ci was not running anymore, I updated the configuration so now it should raise an issue 🙂

Reply:
Indeed, that's been updated!